app/helpers/samples_helper.rb

```diff
@@ -262,13 +260,12 @@ def sample_uploader(sample)
     user
   end
 
-  def sample_derived_data(sample)
+  def sample_derived_data(sample, pipeline_run)
```

Comment:
actually this won't make a difference. 
if you call sample.pipeline_runs.first on the sample sample multiple times, the second time and on it the results will be cached.